utils/config/examples/daos_server_unittests.yml

```diff
@@ -0,0 +1,69 @@
+# Example configuration file using sockets
+
+name: daos_server           # map to -g daos_server
+port: 10001                 # mgmt port
+provider: ofi+sockets       # map to CRT_PHY_ADDR_STR=ofi+sockets
+socket_dir: /tmp/daos_sockets
+control_log_mask: DEBUG
+control_log_file: /tmp/daos_control.log
+
+# single server instance per config file for now
+servers:
+-
+  targets: [0-0]            # map to -t 1, just count number of cpus in the range for now
+  fabric_iface: eth0        # map to OFI_INTERFACE=lo
```

Comment:
should this be lo ?  It doesn't match the comment

Reply:
Usually when we run on a single node, we use lo with sockets because it's always there and eth0 may not be.

Reply:
will apply this fix on follow patch in this commit https://github.com/daos-stack/daos/pull/466/commits/e897f01a14d1e4c517ddcbc32c6cbd9273871ac6 , thanks @jolivier23